.travis.yml

```diff
@@ -85,3 +92,11 @@ matrix:
       os: linux
       compiler: clang
       before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DUNDEFINED_BEHAVIOR_SANITIZER=TRUE
+
+    # Old Version GCC 4.4
+    - name: "Linux GCC 4.4 Build"
+      os: linux
+      compiler: gcc
+      before_script:
+        - export CC=gcc-4.4 && export CXX=g++-4.4 && mkdir build && cd build && cmake .. -DBUILD_TEST=FALSE
```

Comment:
You don't have to set `BUILD_TEST=FALSE` it should be off by default!

If not mind turning it off by default? I don't want users to be force to build C++ code if they don't have to!

Reply:
Will remove in a separate PR.